gui/builtinShipBrowser/fitItem.py

```diff
@@ -17,10 +17,29 @@
 from gui.bitmap_loader import BitmapLoader
 from gui.builtinShipBrowser.pfBitmapFrame import PFBitmapFrame
 from service.fit import Fit
-
+import gui.fitCommands as cmd
 pyfalog = Logger(__name__)
 
 
+class FitRenameCommand(wx.Command):
```

Comment:
Need to move this, this was the very first test case of the command stuff, and I just stuck it here. >_>